posixlib/src/main/resources/scala-native/unistd.c

```diff
@@ -1,6 +1,10 @@
 #include <unistd.h>
 #include "types.h"
 
+extern char **environ;
```

Comment:
I did not check this file against the posix standard, but it looks better than what it
replaces.

Reply:
Yes, this is better.